.github/workflows/ci.yml

```diff
@@ -72,6 +71,28 @@ jobs:
           cd ..
           echo "LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$PWD/usr/lib" >> $GITHUB_ENV
           echo "TOOLCHAIN_OPTION=$TOOLCHAIN_OPTION -DCMAKE_PREFIX_PATH=$PWD/usr" >> $GITHUB_ENV
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Set up Docker Buildx
```

Comment:
Does Buildx really help us somehow? Just asking, because I have not used/switched to it (yet).

Reply:
I was trying `docker/setup-buildx-action@v1` as it seems the `docker/build-push-action@v2` works in a container meaning that the image is not available to the following steps and using `docker/setup-buildx-action@v1` is listed as the way to get it to not run in a container. `load: true` should have handled that already, but it didn't seem to be working.